ADOPTERS.md

```diff
@@ -0,0 +1,15 @@
+# Octant Adopters
+
+## Success Stories
+
+Below is a list of organizations that are using Octant to gain insight on their Kubernetes deployments and have publicly shared the details of how they use it.
+
+
+## Solutions built with Octant
+
+Below is a list of plugins created to extend the Octant feature set for additional use cases.
```

Comment:
We also have a place to list plugins over in our `reference.octant.dev` website and I believe we talked about listing plugins under the `Community` section of the main `octant.dev` website. Is there a plan to reduce these down to a single location?

Is this document going to be the location? Or will this just link out to one of those other places?